Multiple email has to be separated by comma instead of spaces

saravankumarpa opened the pull request:
Multiple email has to be separated by comma instead of spaces

*Issue #:*
https://github.com/bcgov/entity/issues/2260

*Description of changes:*

*Checklist:*
I confirm that below checklist items are addressed in this pull request; (check the boxes applicable)
- [ ] No lint errors
- [ ] No test case failures and proper coverage for all modules/classes
- [ ] Updated the deployment configs for new environment variable(s)
- [ ] Updtaed the postman collection in entity repository (https://github.com/bcgov/entity/tree/master/api-e2e/postman) for e2e tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

sutherlanda commented:
What was the root cause of the bug addressed by this?  The original ZenHub issue is unclear.

saravankumarpa commented:
> What was the root cause of the bug addressed by this? The original ZenHub issue is unclear.

Added the reply.
This got introduced when we switched to the notification service.The old mail service accepted spaces , but the new mail notify service needed commas instead

sutherlanda commented:
> > What was the root cause of the bug addressed by this? The original ZenHub issue is unclear.
> 
> Added the reply.
> This got introduced when we switched to the notification service.The old mail service accepted spaces , but the new mail notify service needed commas instead

Ah okay, thanks for clarifying @saravankumarpa

saravankumarpa commented:
> > > What was the root cause of the bug addressed by this? The original ZenHub issue is unclear.
> > 
> > 
> > Added the reply.
> > This got introduced when we switched to the notification service.The old mail service accepted spaces , but the new mail notify service needed commas instead
> 
> Ah okay, thanks for clarifying @saravankumarpa

I hope it fixes. Not 100% sure the bug. But there is a definite bug here which we have to fix
